customers.js

```diff
@@ -0,0 +1,90 @@
+const createCustomer = async (pool, {fullName, email, password }) => {
+    const {rows:[customer]} = await pool.query(`
+        INSERT INTO customers (
+            full_name,
+            email,
+            password
+        ) 
+        VALUES ($1, $2, $3)
+        RETURNING *;   
+    `, [fullName, email, password]);
+    return customer;
+}
+
+const getAllCustomers = async (pool) => {
+    const {rows: customers} = await pool.query(`
+        SELECT * FROM customers;
+    `)
+    return customers;
+}
+
+const getCustomerLinkedToOrder = async (pool, orderId) => {
+    const {rows:[order]} = await pool.query(`
+        SELECT * FROM orders
+        WHERE id = $1;
+    `, [orderId])
+    if (order === undefined) {
+        throw new Error(`Could not retrieve data because id provided (${id}) does not exist in table.`)
+    } 
+
+    const {rows:[customer]} = await pool.query(`
+        SELECT * FROM customers
+        WHERE id = $1;
+    `, [order["customer_id"]])
```

Comment:
Fairly certain you could achieve this without 2 separate queries if you use a `JOIN`

Reply:
Did it!